src/sentry/api/urls.py

```diff
@@ -973,6 +974,12 @@
         DataExportDetailsEndpoint.as_view(),
         name="sentry-api-0-organization-data-export-details",
     ),
+    # Feedback
+    re_path(
```

Comment:
we should remove organization slug from this route -- can grab the organization from the project if we need it, as per @billyvg

Reply:
and given that we should probably rename the endpoint for ingest `OrganizationFeedbackIngestEndpoint` as it will be a different endpoint from one that we use for search. any thoughts on naming @cmanallen ?

Reply:
+1

Reply:
Cc @c298lee